scripts/qa_crowbarsetup.sh

```diff
@@ -4208,6 +4208,69 @@ EOF
     exit $ret
 }
 
+function oncontroller_testpreupgrade
+{
+    . .openrc
+
+    # create cirros image
+    openstack image create --disk-format qcow2 --container-format bare --public --location http://download.cirros-cloud.net/0.3.4/cirros-0.3.4-x86_64-disk.img --os-image-api-version 1 cirros
```

Comment:
I only found cirros/docker image in this script. The rest seem to be related to jenkins scripts.